src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html

```diff
@@ -409,6 +415,18 @@ <h4>Are you certain you want to delete [[removeAmount]] seat(s) from the followi
             this.$.seatDialog.open();
           }
 
+          assignSeat() {
+            const seat = this.seatToEdit;
+            const assignId = this.assignId;
```

Comment:
Why not give the variable the action name? `ticketID`

Reply:
it has been changed!